packages/devops/scripts/lambda/actions/delete_old_deployments.py

```diff
@@ -0,0 +1,20 @@
+import time
+from datetime import datetime
+from helpers.utilities import find_instances, get_tag
+from helpers.teardown import teardown_instance
+
+# Deletes all deployments with an expired "DeleteAfter" tag
+
+def delete_old_deployments(event):
+    current_datetime = datetime.now()
+    current_date_and_hour = time.strftime("%Y-%m-%d %H")
```

Comment:
What I'm going for here with the filter is limiting the number of instances that will match and have their tag checked, both for efficiency and also because we don't want to go over 1 page of instances or I'd have to and in pagination handling (though just looking it up now, the default max is [probably 1000](https://stackoverflow.com/a/58442322/1988532))

Reply:
Yeah I think that we're never going to have 1000 ready for deletion on the same day, so I'll pull it back to date only for clarity

Reply:
Not fussed. I saw some stuff below where hour makes sense. Happy either way!